### PR TITLE
Add Quick Summary of RFC process to README

### PR DESCRIPTION
**Pull request by jackkoenig**

### CODE REVIEW

**Review by ucbjrl — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -43,6 +44,16 @@ language is evolving in.
...
 
+## Quick Summary
+[Quick Summary]: #quick-summary
+
+  - Fork this repo
+  - Copy `0000-template.md` to `text/0000-my-feature.md` (don't assign a number yet)
+  - Fill out the template with your proposed change
+  - Make a Pull Request on this repo
+  - Your PR will be reviewed by the relevant sub-project team
+  - Your proposal may be "accepted", "rejected", or "postponed"
```
Comment (jackkoenig) [inline README.md:55]: Expand on what these mean

**Comment by jackkoenig** [comment]
Need to add notion of "final approval"

`acceptance` is approval of an RFC with no implementation obligation, once someone actually implements the RFC, that code itself must be approved and merged, thus "final approval".

**Comment by jackkoenig** [comment]
Better term than "final approval" is "implemented" or maybe "upstreamed"

**Pull request merged**